corehq/apps/geospatial/reports.py

```diff
@@ -88,6 +89,44 @@ def _get_geo_location(self, case):
     def rows(self):
         raise NotImplementedError()
 
+    @property
+    def disbursement_parameters(self):
+        try:
+            config = GeoConfig.objects.get(domain=self.domain)
+        except GeoConfig.DoesNotExist:
+            return []
+
+        return self._construct_disbursement_parameters(config)
+
+    @staticmethod
+    def _construct_disbursement_parameters(config):
+        def get_parameter_spec(name, value, unit=None):
```

Comment:
The parameters shown is based on the config settings and does not necessarily show the actual parameters the disbursement is run with. This is particularly the case for the "max cases per user" setting, since if it's unspecified, the algorithm determines what this value should be on the fly (number of cases divided by number of users)

Reply:
@mkangia I haven't thought of that workflow, but I think that's a brilliant idea!

Reply:
@mkangia See https://github.com/dimagi/commcare-hq/pull/34716/commits/49db7c0122a6f60ddc8ca45e2aa4d35557f255db.